docs/reference/ingest/ingest-node.asciidoc

```diff
@@ -1135,6 +1135,51 @@ In this example, if the `remove` processor does fail, then
 the array elements that have been processed thus far will
 be updated.
 
+
+Another example, using the attachment processor:
```

Comment:
I think we should move this to the attachment processor doc instead.
Then write here `Another advanced example can be found in {plugins}/using-ingest-attachment.html[attachment processor documentation]`.

WDYT?